it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -0,0 +1,157 @@
+        public static final int DEFAULT_WAIT_POLL_ITER = 300;
+
+        // Default time to wait before next layout poll: 1 second
+        public static final int DEFAULT_WAIT_TIME = 1;
+    }
+
+    /**
+     * Common configuration for Universe initialization
+     */
+    class UniverseFixture extends AbstractUniverseFixture<UniverseParams> {
+
+        @Override
+        public UniverseParams data() {
+            this.servers = FixtureUtil.buildMultipleServers(numNodes, corfuCluster.getName());
+            servers.forEach(corfuCluster::add);
+
+            return UniverseParams.universeBuilder()
+                    .build()
+                    .add(corfuCluster);
+        }
+    }
+
+    /**
+     * Configuration for VM specific Universe initialization
+     */
+    class VmUniverseFixture extends AbstractUniverseFixture<VmUniverseParams> {
+        private static final String VM_PREFIX = "corfu-vm-";
+
+        @Override
+        public VmUniverseParams data() {
+            this.servers = FixtureUtil.buildVmMultipleServers(numNodes, corfuCluster.getName(), VM_PREFIX);
+            servers.forEach(corfuCluster::add);
+
+            ConcurrentMap<String, String> vmIpAddresses = new ConcurrentHashMap<>();
+            for (int i = 0; i < numNodes; i++) {
+                vmIpAddresses.put(VM_PREFIX + (i + 1), "0.0.0.0");
+            }
+
+            Properties credentials = new Properties();
+            try (InputStream is = ClassLoader.getSystemResource("vm.properties").openStream()) {
+                credentials.load(is);
+            } catch (IOException e) {
+                throw new IllegalStateException("Can't load credentials", e);
+            }
+
+            VmUniverseParams params = VmUniverseParams.builder()
+                    .vSphereUrl("https://10.173.65.98/sdk")
+                    .vSphereUsername(credentials.getProperty("vsphere.username"))
+                    .vSpherePassword(credentials.getProperty("vsphere.password"))
+                    .vSphereHost("10.172.208.208")
+                    .templateVMName("IntegTestVMTemplate")
+                    .vmUserName("vmware")
+                    .vmPassword("vmware")
+                    .vmIpAddresses(vmIpAddresses)
+                    .networkName("corfu_network")
+                    .build();
+            params.add(corfuCluster);
+
+            return params;
+        }
+    }
+
+    @Data
+    @Accessors(chain = true)
+    abstract class AbstractUniverseFixture<T extends UniverseParams> implements Fixture<T> {
+        protected int numNodes;
+        protected ClientParams client;
+        protected CorfuClusterParams corfuCluster;
+        protected ImmutableList<CorfuServerParams> servers;
+
+        public AbstractUniverseFixture() {
+            this.numNodes = 3;
+            this.client = ClientParams.builder().build();
+            this.corfuCluster = CorfuClusterParams.builder().build();
+        }
+    }
+
+    class FixtureUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)